### DISCUSSION TITLE
Win10: Jamulus client hangs on startup

### DISCUSSION THREAD

**Discussion by marcblum-ac** [jamulussoftware/jamulus#1584] [Technical Support & Help]
I'm new to Jamulus.
OS: Windows 10 pro
audio device: SPL Creon with respective ASIO driver
DAW: Reaper

Issue on install: after install a box poppud up, telling me that the ASIO driver should provide 48kHz, but does 44,1 kHz, and should be configured correctly. That irritated me, as I'm working with 48 kHz. I clicked it away to investigate later. But then nothing more happened. jamulus.exe just hangs in the background.

Recurring issue on startup: jamulus.exe just hangs in the background.

I tried different cleanups/re.installs, as user providing admin credentials, directly as admin. No resolution. Jamulus.exe keeps hanging in the background.

No idea where to find logs. In install folder and within \app data I found nothing.

Pls advise.

**Comment by npostavs** [comment]
Do you mean you got messages like this? That is simply the current behaviour of Jamulus, if it fails to setup the audio you can't use it all (although the part where it fails to exit properly is a known bug (#872) which is fixed in the development version). 
![image](https://user-images.githubusercontent.com/287742/116766517-f017b080-a9f8-11eb-9c1b-c8e53267486b.png)
![image](https://user-images.githubusercontent.com/287742/116766455-a62eca80-a9f8-11eb-8bb9-27308f958d61.png)

I'm working on version that will not die completely when there is an audio error (#1378), so that "investigate later" will be an option. You can get it here if you want to try: <https://github.com/jamulussoftware/jamulus/suites/2599353514/artifacts/56836492>. However, you'll need to resolve the sample rate issue if you want to get any sound.

**Comment by marcblum-ac** [comment]
Yes, that's the issue.
In the meantime I found a place in the Windows sound setting provoking to request 44,1kHz. Changed it to 48kHz.

I cleaned my machine and reinstalled the dev version.
Now it shows below error message which is what I expected. I can now continue investigating Windows and ASIO driver settings.

Thanks for the help!

![grafik](https://user-images.githubusercontent.com/77191126/116771454-d81d4c80-aa4b-11eb-963f-367cc533a8f7.png)
